CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changes to Mapbox Directions for Swift
 
+## v2.6.1
```

Comment:
This release is needed to fix build issues in https://github.com/mapbox/mapbox-navigation-ios/pull/4274.